CONTRIBUTING.md

```diff
@@ -1,109 +1,62 @@
-Contributing to Zeppelin
+Contributing to Open Zeppelin
 =======
 
-## Design Guidelines
+We really appreciate and value contributions to Open Zeppelin. Please take 5' to review the items listed below to make sure that your contributions are merged as soon as possible. 
 
-These are some global design goals in Zeppelin.
+## Contribution guidelines
 
-### D0 - Security in Depth
-We strive to provide secure, tested, audited code. To achieve this, we need to match intention with function. Thus, documentation, code clarity, community review and security discussions are fundamental.
+Smart contracts manage value and are highly vulnerable to errors and attacks. We have very strict guidelines, please make sure to review them: ["Contribution guidelines wiki entry"](https://github.com/OpenZeppelin/zeppelin-solidity/wiki/Contrbuting-guidelines).
 
-### D1 - Simple and Modular
-Simpler code means easier audits, and better understanding of what each component does. We look for small files, small contracts, and small functions. If you can separate a contract into two independent functionalities you should probably do it.
+## Branching
 
-### D2 - Naming Matters
+When it comes to branching, we follow the "git-flow" method. Please see ["Git flow wiki entry"](https://github.com/OpenZeppelin/zeppelin-solidity/wiki/Git-flow).
 
-We take our time with picking names. Code is going to be written once, and read hundreds of times. Renaming for clarity is encouraged.
+*IMPORTANT* This means that our main development branch is `development`, not `master`.
 
-### D3 - Tests
+## Creating Pull Requests (PRs)
 
-Write tests for all your code. We encourage Test Driven Development so we know when our code is right. Even though not all code in the repository is tested at the moment, we aim to test every line of code in the future.
+As a contributor, you are expected to fork this repository, work on your own fork and then submit pull requests. The pull requests will be reviewed and eventually merged into the main repo. See ["Fork-a-Repo"](https://help.github.com/articles/fork-a-repo/) for how this works.
 
-### D4 - Check preconditions and post-conditions
+*IMPORTANT* Please use `rebase` instead of `merge`.
```

Comment:
Add "when updating your branch to include the latest commits", or something similar that explains in what scenario the contributor is expected to rebase.